normandy/recipes/tests/test_serializers.py

```diff
@@ -28,6 +29,117 @@ def test_it_works(self, rf):
             'is_approved': recipe.is_approved
         }
 
+    # If the action specified cannot be found, raise validation
+    # error indicating the arguments schema could not be loaded
+    def test_validation_with_wrong_action(self):
+        serializer = RecipeSerializer(data={
+            'action': 'action-that-doesnt-exist', 'arguments': {}
+        })
+
+        with pytest.raises(serializers.ValidationError):
+            serializer.is_valid(raise_exception=True)
+
+        assert serializer.errors['arguments'] == ['Could not find arguments schema.']
+
+    # If the action can be found, raise validation error
+    # with the arguments error formatted appropriately
+    def test_validation_with_wrong_arguments(self):
+        ActionFactory(
+            name='show-heartbeat',
+            arguments_schema={
```

Comment:
maybe pull this out into something like:
```python
ARGUMENTS_SCHEMA = {
    "required": ["surveyId", "surveys"],
    "properties": {
        "surveyId": {"type": "string"},
        "surveys": {
            "type": "array",
            "minItems": 1,
            "items": {
                "properties": {
                    "title": {"type": "string"},
                    "weight": {"type": "integer", "minimum": 1}
                },
                "required": ["title", "weight"]
            },
        },
    },
}
```
at the top of the file or in `__init__.py`.